addons/account_debit_note/wizard/account_debit_note_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_account_debit_note" model="ir.ui.view">
+            <field name="name">account.debit.note.form</field>
+            <field name="model">account.debit.note</field>
+            <field name="arch" type="xml">
+                <form string="Create Debit Note">
+                    <field name="move_type" invisible="1"/>
+                    <group>
+                         <group>
+                             <field name="reason"/>
+                             <field name="date" string="Debit Note Date"/>
+                             <field name="copy_lines" attrs="{'invisible': [('move_type', 'in', ['in_refund', 'out_refund'])]}"/>
+                         </group>
+                         <group>
+                             <field name="journal_id"/>
```

Comment:
@jco-odoo for mi this journal could be removed to make it simpler. but if we keep it, we should add a domain considering the right company or the right journal (sale or purchase), actually you can choose any journal.

Reply:
Hi @jco-odoo , regarding @jjscarafia comment, this is something that needs to be fixed in standard because even in the standard Credit Note wizard this field has the same behavior.